tests/integration-tests/src/test/kotlin/config/services/Agent.kt

```diff
@@ -0,0 +1,37 @@
+package config.services
+
+import com.sksamuel.hoplite.ConfigAlias
+import org.testcontainers.containers.ComposeContainer
+import org.testcontainers.containers.wait.strategy.Wait
+import java.io.File
+
+data class Agent(
+    val version: String,
+    @ConfigAlias("http_port") val httpPort: Int,
```

Comment:
I would think about using the same camel case in the config files, so @ConfigAlias wouldn't be required

Reply:
It is preferred to have camelCase for Kotlin, so, I've decided to do like that.

Reply:
I'm proposing to use camelCase in the configuration files as well